sass/routes/_backstage.scss

```diff
@@ -251,6 +251,23 @@
     }
 }
 
+.item-actions {
+    button,
+    a {
```

Comment:
please don't target html tags directly... use a class or id (preferred if possible) to increase specificity

Reply:
missed that too 🙈